urllib3/connectionpool.py

```diff
@@ -695,7 +696,7 @@ def urlopen(self, method, url, body=None, headers=None, retries=None,
                 release_conn=release_conn, **response_kw)
 
         # Check if we should retry the HTTP response.
-        if retries.is_forced_retry(method, status_code=response.status):
+        if retries.is_retry(method, response.status, bool(response.getheader('Retry-After'))):
```

Comment:
I don't think there's any need to squish all of this into one line, we can split the `bool` line out into a temporary variable.